src/Nethermind/Nethermind.JsonRpc/Modules/DebugModule/DebugBridge.cs

```diff
@@ -77,6 +80,11 @@ public int DeleteChainSlice(long startNumber)
             return _blockTree.DeleteChainSlice(startNumber);
         }
 
+        public Task<bool> SetReceiptMigrationPoint(long blockNumber)
```

Comment:
rename to run migration

Reply:
Not sure about that. It does what it does, sets migration point, and then migrations runs or not depending if it needs to run.